backend/content/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('trending/', views.content_trending),
+    path('search/<str:search_str>/', views.content_search),
+    path('<int:user_id>/recommendation/', views.content_recommendation),
+    path('<int:user_id>/favorite/', views.user_favorite_list),
+    path('<int:user_id>/favorite/<int:content_id>/', views.content_favorite),
+    path('<int:content_id>/review/', views.review_content),
```

Comment:
I think this should be at backend/review

Reply:
@cw10108 That api starts with /content/, so i put that in content/urls we can modify it later then.